### ISSUE TITLE
Translations missing for group issue page

### ISSUE THREAD

**Issue by mvl22** [cyclestreets/cyclescape#342]
Issue editing, voting, etc. are missing translations:

    translation missing: en-GB.group.issues.vote_up.success

e.g. at http://camcycle.cyclescape.org/issues/1472

**Comment by smsm1** [comment]
I've already got the fix for this implemented, it just needs to be pulled across to master: https://github.com/cyclestreets/cyclescape/commit/89a256a52dc4b35164c6639b4017f70843e4b87d

**Comment by mvl22** [comment]
Thanks. OK, that's blocked on the other issue commented there in the same branch. If you can split this into a new branch then obviously I could bring it in sooner.

**Comment by nikolai-b** [comment]
Committed in master as https://github.com/cyclestreets/cyclescape/commit/310ebcffd8831a6a36dd977475cc46b3ddc4f159
